src/condor_schedd.V6/qmgmt.cpp

```diff
@@ -5728,7 +5729,6 @@ AddSessionAttributes(const std::list<std::string> &new_ad_keys)
 				SetAttribute(job.cluster, job.proc, ATTR_USER, "UNDEFINED");
 			}
 		}
```

Comment:
This code was disabled for a reason, it does not belong here and besides the User attribute of jobs is always set anyway, and it is not ok to call SetAttribute at this point because this function is called during CommitTransaction.

Reply:
Can you provide a reproducer where the `User` attribute is set?  Neither I nor Nebraska could find a way to get `User` attribute to be set by `condor_submit` when `USER_IS_THE_NEW_OWNER=true`;  without `User` attributes set, effectively nothing works.

Here's the reproducer; set `USER_IS_THE_NEW_OWNER=true` and bounce the schedd before testing:
```
Executable = test.sh
Arguments = 5
Universe = vanilla
output = test.out.$(Cluster)
error = test.err.$(Cluster)
Log = test.log.$(Cluster)
transfer_input_files = upfile.test
Queue 1
```

Not sure why it's not OK to call `SetAttribute` here given `SetSecureAttribute` is used heavily in this function.